roles/multi-tenancy/tasks/onboard-offboard-cas-servers.yaml

```diff
@@ -52,74 +52,11 @@
   tags:
     - offboard
 
-# Delete the site.yaml file
-- name: delete site.yaml
+# Delete the sasdeployment.yaml file
+- name: delete sasdeployment.yaml
```

Comment:
Checking for clarity. This is the CRD file that's generated for the orchestration tooling that replaces site.yaml file?

Reply:
Yes that is correct.